volttrontesting/utils/platformwrapper.py

```diff
@@ -134,9 +134,11 @@ def build_vip_address(dest_wrapper, agent):
 def start_wrapper_platform(wrapper, with_http=False, with_tcp=True,
                            volttron_central_address=None,
                            volttron_central_serverkey=None,
-                           add_local_vc_address=False):
+                           add_local_vc_address=False,
+                           instance_name='volttron1'):
```

Comment:
For tests aren't we using volttron_test as instance name? I don't want to use volttron1 as it is the default in the environment outside of pytest too.

Reply:
done